### DISCUSSION TITLE
por favor, corrige el error de una misión

### DISCUSSION THREAD

**Discussion by wowtomycito** [chromiecraft/chromiecraft#2637] [General]
hola. hay una misión que está mal y hace algunos días y nada, para q vea las imágenes
![1](https://user-images.githubusercontent.com/96457130/146866390-c88764f4-a58e-4084-99aa-7205d7353962.jpg)
![2](https://user-images.githubusercontent.com/96457130/146866399-13919f9d-1f1c-488b-b046-5528a463787b.jpg)

**Comment by Amber-Haze** [comment]
Buenas. El mineral que te piden aparece en los mismos nodos que los otros (hay que minarlos para que aparezca la sangrita), pero aun asi tarda mucho en aparecer.
El bug está reportado ya
https://github.com/chromiecraft/chromiecraft/issues/634
